x-pack/plugins/secops/public/components/drag_and_drop/helpers.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DropResult } from 'react-beautiful-dnd';
+import { Dispatch } from 'redux';
+import { ActionCreator } from 'typescript-fsa';
+
+import { timelineActions } from '../../store';
+import { dragAndDropActions } from '../../store/local/drag_and_drop';
+import { IdToDataProvider } from '../../store/local/drag_and_drop/model';
+import { DataProvider } from '../timeline/data_providers/data_provider';
+
+export const draggableContentPrefix = 'draggableId.content.';
+
+export const droppableContentPrefix = 'droppableId.content.';
+
+export const droppableTimelineProvidersPrefix = 'droppableId.timelineProviders.';
+
+export const draggableTimelineFlyoutButtonPrefix = 'draggableFlyoutButton';
+
+export const droppableTimelineFlyoutButtonPrefix = 'droppableFlyoutButton';
```

Comment:
These will probably be fixed with the bug fixes coming next on the last D&D bug fixes. Just mentioning you are leaving them as a shortened name here in case you don't want that for this PR.